DQMOffline/Trigger/python/SusyMonitoring_cff.py

```diff
@@ -1,7 +1,21 @@
 import FWCore.ParameterSet.Config as cms
 
 from DQMOffline.Trigger.RazorMonitor_cff import *
+from DQMOffline.Trigger.TopMonitor_cfi import hltTOPmonitoring
+
+susyMuEGMonitoring = hltTOPmonitoring.clone()
+susyMuEGMonitoring.FolderName = cms.string('HLT/SUSY/MuonEG/')
+susyMuEGMonitoring.nmuons = cms.uint32(1)
+susyMuEGMonitoring.nphotons = cms.uint32(1)
+susyMuEGMonitoring.nelectrons = cms.uint32(0)
+susyMuEGMonitoring.njets = cms.uint32(0)
+susyMuEGMonitoring.enablePhotonPlot =  cms.bool(True)
+susyMuEGMonitoring.muoSelection = cms.string('pt>26 & abs(eta)<2.1 & isPFMuon & isGlobalMuon & isTrackerMuon & numberOfMatches>1  & innerTrack.hitPattern.trackerLayersWithMeasurement>5 & innerTrack.hitPattern.numberOfValidPixelHits>0  & globalTrack.hitPattern.numberOfValidMuonHits>0 & globalTrack.normalizedChi2<10 & (pfIsolationR04.sumChargedHadronPt + max(pfIsolationR04.sumNeutralHadronEt + pfIsolationR04.sumPhotonEt - (pfIsolationR04.sumPUPt)/2.,0.) )/pt<0.15') 
```

Comment:
this level of complexity might cause problems - could you double check that the RSS of an example job does not change much with and without this module running?